frontend/javascripts/oxalis/view/plane_view.js

```diff
@@ -137,6 +143,69 @@ class PlaneView {
     }
   }
 
+  performIsosurfaceHitTest(): ?THREE.Vector3 {
```

Comment:
Did you experiment with throttling this function? To me, during testing, the TD view felt a little sluggish and I was wondering whether this could help to make it feel more performant :) I do understand that the "instantness" of the hover effect contributes to the "coolness" of the future and I do want to keep that, so maybe just throttling it a little is enough.
It's not a blocker though as the performance hit is only noticeable when isosurfaces are loaded.

Reply:
You are absolutely right. I definitely needed to add throttling, as it doesn't make sense to compute this on every render. I opted for 150 ms as a throttling delay, which felt like a good trade-off between performance and delay to me.